conda-requirements

```diff
@@ -11,4 +11,3 @@ numexpr==2.4.4
 Cython==0.21
 networkx==1.10
 pytest==2.6.3
-pytest-html==1.8.1
```

Comment:
You are doing a change here without providing a 'backup'. This is generally bad.

It is correct, that we consider `conda-requirements` obsolete but as long as they are included in our repository we should keep them updated, too.